### PR TITLE
Fix Travis CI build issue

### PR DESCRIPTION
**Pull request by damcclean**
This pull request fixes an error that would occur when running the test suite on Travis CI.

I've removed the line that installs the `apcu` in the `travis.yml` file, Travis' config file.

This PR addresses #300.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@ env:
     - FLAGS=""
 
 before_install:
-  - echo yes | pecl upgrade apcu
   - |
```
Comment (ankitpokhrel) [inline .travis.yml:15]: I think we can also remove line 16 to 19

**Review by ankitpokhrel — COMMENTED** [review]
Hi @damcclean, thank you for the PR. I added one small comment, can you check it please.

File: .travis.yml (hunk 1 above)
Comment (damcclean) [reply .travis.yml]: I've removed the requested lines.

**Comment by damcclean** [comment]
Sorry to bother you, could you add the `hacktoberfest` label to this repo (without the 2020) so it can count towards my PRs for the month 😅

**Pull request merged**